javascript/javascript2/week2/hyfBayhelpers/script.js

```diff
@@ -0,0 +1,105 @@
+window.getAvailableProducts = function() {
```

Comment:
Only when you are free 😀😀😀😀 The code is fine. The nested function is fine for the 'factory' function
1. read about a modular way of adding function
2. try object > method and try to execute.